### DISCUSSION TITLE
Is there a way to maximize current pane?

### DISCUSSION THREAD

**Discussion by OriLiMu** [zellij-org/zellij#3320] [Q&A] [answered]
I search for the doc not found related

**Comment by Lockszmith-GH** [comment]
Short answer: `Ctrl p` (to enter **pane** mode), and then `f` (to toggle the focused pane fullscreen mode)

See below on how I search the docs when looking for this.

**Reply by Lockszmith-GH** [reply, marked as answer]
Start by identifying the action, in this case it is the [ToggleFocusFullscreen](https://zellij.dev/documentation/keybindings-possible-actions.html?highlight=ToggleFocusFullscreen#togglefocusfullscreen) action.

The docs don't necessarily cover ALL keybindings, (althogh this one has a [mention](https://zellij.dev/documentation/keybindings-binding.html?highlight=ToggleFocusFullscreen#binding-keys)).

I personally find the easiest way to investiage available key-bindings is to [dump the default config](https://zellij.dev/documentation/configuration.html?highlight=dump#how-to-dump-the-default-configuration-to-stdout) and inspect it.

You should find that `ToggleFocusFullscreen` action is under the `pane` mode:

```kdl
keybinds {
...
    pane {
        ...
        bind "f" { ToggleFocusFullscreen; SwitchToMode "Normal"; }
        ...
    }
...
    shared_except "pane" "locked" {
        bind "Ctrl p" { SwitchToMode "Pane"; }
    }
...
```

This means with `Ctrl p` you enter `pane` mode, and with `f` you toggle the FullScreen on the Focused pane.

**Comment by giyany** [comment]
Ctrl - p + f will take the pane to fullscreen,
Ctrl - t + b will break the pane into a new tab.
